utils.py

```diff
@@ -211,62 +211,109 @@ def saveImage(window):
     overlays = window.overlays
     exthandler = window.exthandler
     Tk().withdraw()
-    file = filedialog.asksaveasfile(
-        filetypes=[("PNG Image", "*.png")], defaultextension=".png"
+    file = filedialog.asksaveasfilename(
+        filetypes=[("PNG Image", "*.png"),("TIFF Image", "*.tiff"),("CSV File", "*.csv")]
     )
 
     if file:
-        filename = file.name
-
-        if (
-            messagebox.askquestion(
-                "Save options", "Do you want to save with the annotations?"
+        filename = file.split("/")[-1]
+
+        if filename.endswith('.tiff'):
+            if (messagebox.askquestion("Before we proceed",f"Are you sure you want to export image as {filename}?") == "yes"):
```

Comment:
@Lemorra - can you mention in this message that the TIFF will be saved with Kelvin values, and not a false colour mapping?

Reply:
Done.